packages/ubuntu_desktop_installer/lib/pages/installation_slides/installation_slides_model.dart

```diff
@@ -47,7 +48,7 @@ class InstallationSlidesModel extends ChangeNotifier with SystemShutdown {
 
   void _updateStatus(ApplicationStatus? status) {
     if (state == status?.state) return;
-    print(
+    appLogger.debug(
         'Subiquity state: ${_formatState(state)} => ${_formatState(status?.state)}');
```

Comment:
now that i think of this, we should probably output subiquity states `subiquity_client.dart` instead :)

Reply:
This is what we have in the current main:
```
flutter: Subiquity state: UNKNOWN => RUNNING
flutter: DEBUG subiquity_client: GET http://localhost/meta/status?cur=%22RUNNING%22
flutter: Subiquity state: RUNNING => POST_RUNNING
flutter: DEBUG subiquity_client: GET http://localhost/meta/status?cur=%22POST_RUNNING%22
flutter: Subiquity state: POST_RUNNING => UU_RUNNING
flutter: DEBUG subiquity_client: GET http://localhost/meta/status?cur=%22UU_RUNNING%22
flutter: Subiquity state: UU_RUNNING => DONE
```

What if we printed all subiquity responses? (Shortened to max XX chars?) Then we could throw print this out.

Reply:
I moved the logging of state changes to `subiquity_client`. We can improve/systematize logging of subiquity responses in a follow-up PR, if that works for you.